Sources/Trickery/HexagonParameters.swift

```diff
@@ -0,0 +1,44 @@
+import CoreGraphics
+
+struct HexagonPamaters {
```

Comment:
I don't think this makes sense to be in Trickery? Seems like it's more for the Demo's purposes and users of our library do not need it.

Reply:
`HexagonPamaters` => `HexagonParameters`

Reply:
Looking above this might be duplicated

Reply:
It shouldn't I removed it as soon I saw it